.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Your issue may already be reported! Please check [issue tracker](../) before creating a new one.
```

Comment:
```suggestion
---
name: Bug report
about: Report a bug.
title: ''
labels: 'bug'
assignees: ''
---

Your issue may already be reported! Please check [issue tracker](../) before creating a new one.
```

iirc we can indicate that it's a bug report like this and automatically assign the bug label.